src/index.html

```diff
@@ -39,7 +39,11 @@
   <body>
     <!-- W taki sposbu dodajemy fragmenty w główny HTML-plik strony. -->
     <include src="./partials/Hero header.html"></include>
+    <include src="./partials/franchais-modal.html"></include>
+    <include src="./partials/location-modal.html"></include>
     <!-- Skrypt jest na samym dole kodu. On juz ma w sobie podłączony SAAS plik. -->
     <script src="./index.js" type="module"></script>
+    <script src="./js/modal.js"></script>
+    <script src="./js/modal.-location.js"></script>
```

Comment:
w skrypcie modal location chyba błąd w nazwie, powinno być modal-location.js

Reply:
hehe jakoś podpięła się "kropka", ale ogólnie działa nawet z tą nazwą spoko w nastepnym  branchu poprawie

Reply:
wiadomo, będzie działać . To tylko dla poprawności zapisu na później 👍